src/controllers/controller.polarArea.js

```diff
@@ -136,9 +137,13 @@ module.exports = DatasetController.extend({
 			start += angle;
 		}
 
-		helpers.each(meta.data, function(arc, index) {
-			me.updateElement(arc, index, reset);
-		});
+		for (i = 0, ilen = arcs.length; i < ilen; ++i) {
+			arcs[i]._options = me._resolveElementOptions(arcs[i], i, reset);
```

Comment:
`reset` is unnecessary. controller.doughnut.js has the same issue.

Reply:
I fixed this in both the polarArea and doughnut controllers